backend/src/file/file.config.ts

```diff
@@ -0,0 +1,25 @@
+import { Inject, Injectable } from '@nestjs/common';
+import {
+  MulterModuleOptions,
+  MulterOptionsFactory,
+} from '@nestjs/platform-express';
+import * as GridFsStorage from 'multer-gridfs-storage';
+
+Injectable();
+export class GridFsMulterConfigService implements MulterOptionsFactory {
+  gridFsStorage: GridFsStorage;
+  constructor(@Inject('DATABASE_CONNECTION') private readonly db: any) {
+    this.gridFsStorage = new GridFsStorage({
+      db: this.db,
+      file: (_req: any, file: { originalname: string }) => {
```

Comment:
`_req:any` I think it can use type `Request` from `express` lib
and I'd prefer a custom type for `file : { originalname: string }` param `FileRequestType` would be great